apps/feedback/models/question.go

```diff
@@ -0,0 +1,17 @@
+package models
+
+import (
+	"github.com/jinzhu/gorm"
+
+	"github.com/iReflect/reflect-app/db/models/fields"
+)
+
+type Question struct {
+	gorm.Model
+	Text    string       `gorm:"type:text; not null"`
+	Type    int          `gorm:"default:0; not null"`
```

Comment:
Can we create Type to be an enum (backed with int8 is fine) - would be useful in Admin interface?

Reply:
Added todo's to add enum support, will come back to this post basic APIs